### PR TITLE
Docgen Table tag with markers

### PR DESCRIPTION
**Pull request by Dheeraj3reddy**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/overview/document-generation-api/complextableconstructs.md [hunk 1]
```diff
@@ -243,7 +268,75 @@ JSON representation of the input data:
...
+        }
+      ]
+    }
+  ]
+}
+```
+
+![dsfa](../images/nested_tables.png)
+
+Create both tables with their respective table markers. In the above example, the **school** tag acts as the array
```
[the remark this answers is not in the document]
Comment (Abhinandan578) [reply src/pages/overview/document-generation-api/complextableconstructs.md:340]: Also are we not documenting on how to refer outer context and even root context

**Review by Abhinandan578 — CHANGES_REQUESTED** [review]

File: src/pages/overview/document-generation-api/tablewithmarkers.md [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+---
+# Table Tag with markers
+Use Table markers to insert table tags in a document
+
+There are 2 new markers introduced for inserting tables in a document:
+
+- table start ```{% table-start ARRAY_TAG %}```
+- table end   `{% table-end %}`
+
+In the table row that needs to be dynamically expanded, place the table start marker in the first cell and table
```
[the remark this answers is not in the document]
Comment (piyuverma) [reply src/pages/overview/document-generation-api/tablewithmarkers.md:12]: "codified" table-start and "codified" table-end are necessary to be placed in each row (except header row). 
"codified" table-start is always followed by t"codified" able-end in the row or column depending on the table orientation, vertically or horizontally respectively. 
"codified" table-start needs to be placed in the first non-static cell of each row or column depending on the table orientation, vertically or horizontally respectively. 
"codified" table-end needs to be placed in the last non-static cell of each row or column depending on the table orientation, vertically or horizontally respectively.

[the remark this answers is not in the document]
Comment (piyuverma) [reply src/pages/overview/document-generation-api/tablewithmarkers.md:12]: In the "codified" table-start marker, an array field is required, which will be used to create the rows/columns in the table depending on the table orientation, vertically or horizontally respectively.

File: src/pages/overview/document-generation-api/tablewithmarkers.md [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+    }
+
+  ]
+}
+```
+
+![Conditions inside tables with markers](../images/conditions_inside_tables_table_markers.png)
+
+## Nested Tables
+Table tags with markers can also be used inside another table.
```
Comment (piyuverma) [inline src/pages/overview/document-generation-api/tablewithmarkers.md:257]: Nested Tables or Table inside Table is **only** supported using the new Table Markers ("codified"......)

Comment (piyuverma) [reply src/pages/overview/document-generation-api/tablewithmarkers.md:257]: Nested Table should contain its own Table Markers as per the rules defined above -> provide the link of the Rules from above.

Array Field used with "codified" table-start in the **inner/child** table should be the nested Array field used with "codified" table-start in the **outer/parent** table.

File: src/pages/resources/openapi.json [hunk 4]
```diff
@@ -7457,7 +7457,21 @@
           "tr-TR",
           "uk-UA",
           "zh-CN",
-          "zh-TW"
+          "zh-TW",
```
Comment (piyuverma) [inline src/pages/resources/openapi.json:7460]: See this change, what is it about ?

**Review by piyuverma — CHANGES_REQUESTED** [review]

**Pull request closed without merging**